src/features/image/utils/convertToWebp.ts

```diff
@@ -0,0 +1,12 @@
+const convertToWebp = (src: string) => {
+  if (
```

Comment:
Does it make sense to consider that the uppercased name will be passed to an input?

Reply:
Fixed